Runtime/Scripts/RGBotConfigs/RGAction_KeyPress.cs

```diff
@@ -0,0 +1,298 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using RegressionGames.StateActionTypes;
+using RegressionGames.DebugUtils;
+using UnityEngine;
+#if ENABLE_INPUT_SYSTEM
```

Comment:
These currently don't do a whole lot...

Ideally, we can package this class as a separate thing to add on to our SDK so you only pull in dependencies on Unity's inputSystem when you really need to.